AndroidManifest.xml.template

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.unity3d.player"
+    android:installLocation="preferExternal">
+
+    <supports-screens
+        android:anyDensity="true"
+        android:largeScreens="true"
+        android:normalScreens="true"
+        android:smallScreens="true"
+        android:xlargeScreens="true"/>
+
+    <!-- Some mediated networks do not include a manifest with permissions declaration within the aar they distribute. -->
+
+    <!-- The permissions below are required by some of mediated networks. Though they might still work without them, it is not guaranteed -->
+
+    <!-- Required by all mediation networks -->
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <!-- Required by AdColony, AppLovin, (and optionally by FairBid 2) -->
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <!-- Required by AdColony, AdMob, AppLovin, Facebook Audience Network, Tapjoy, UnityAds, Vungle, and ironSource -->
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+
+
+    <!-- Optional permissions. We can leave these permissions out. However, adding them to your project will probably improve ad delivery experience -->
+
+    <!-- Recommended optional permissions required by FairBid 2. -->
+
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+
+    <application
+        android:icon="@mipmap/app_icon"
+        android:label="@string/app_name"
+        android:theme="@style/UnityThemeSelector">
+        <activity
+            android:name="com.unity3d.player.UnityPlayerActivity"
+            android:label="@string/app_name">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+            <meta-data
+                android:name="unityplayer.UnityActivity"
+                android:value="true"/>
+        </activity>
+
+        <!-- Recent versions of AdMob will crash at runtime if the following meta-data tag is not present -->
+        <meta-data
+            android:name="com.google.android.gms.ads.AD_MANAGER_APP"
+            android:value="true"/>
+        <!-- AdMob’s SDK requires publishers to add their AdMob App ID to the app’s AndroidManifest as a child of the tag: -->
+        <meta-data
+            android:name="com.google.android.gms.ads.APPLICATION_ID"
+            android:value="ca-app-pub-xxxxxxxxxxxxxxxx~yyyyyyyyyy"/>
```

Comment:
Let's just use this [doc page](https://developers.google.com/admob/android/quick-start#import_the_mobile_ads_sdk) as reference and keep the 
`android:name="com.google.android.gms.ads.APPLICATION_ID"`
entry only

Reply:
We need this entry because it is required by the unity admob setup
```
If your app uses Ad Manager instead of AdMob, enable Google Ad Manager by clicking Enabled checkbox under Google Ad Manager section.

```
source: https://developers.google.com/admob/unity/quick-start#set_your_admob_app_id

Reply:
No I dont' have any solid argument it was just showing up in Xcode logs and android logs if we don't set the admanager to true in mainifest and info.plist. Setting it to true you won't see the error log asking to set the value.
 
We removed this template file from the PR. We can ignore this.